src/main/java/com/parasoft/findings/jenkins/coverage/ParasoftCoverageMetric.java

```diff
@@ -0,0 +1,26 @@
+package com.parasoft.findings.jenkins.coverage;
+
+import edu.hm.hafner.coverage.Metric;
+import hudson.util.ListBoxModel;
+
+import java.util.NoSuchElementException;
+
+public class ParasoftCoverageMetric {
+    public ListBoxModel getMetricItems() {
+        ListBoxModel options = new ListBoxModel();
+        options.add(getDisplayName(Metric.FILE), Metric.FILE.name());
+        options.add(getDisplayName(Metric.LINE), Metric.LINE.name());
+        return options;
+    }
+
+    public String getDisplayName(Metric metric) {
+        switch (metric) {
+            case FILE:
+                return "File Coverage";
```

Comment:
This display name can be localized. See: `io.jenkins.plugins.coverage.metrics.model.ElementFormatter#getDisplayName(edu.hm.hafner.coverage.Metric)`. Just add the properties entries in the `Messages.properties` file, then right click the `parasoft-findings` module in IDEA Maven view, select **Generate Sources and Update Folders**, the Messages class will be generated and can be used here.